src/System.IO.Ports/tests/Legacy/SerialPort/ctor_str_int.cs

```diff
@@ -3,200 +3,122 @@
-        retValue &= tcSupport.BeginTestcase(new TestDelegate(PHYSICALDRIVE0_14400), TCSupport.SerialPortRequirements.None);
-
-        //[] Error checking for BaudRate
-        retValue &= tcSupport.BeginTestcase(new TestDelegate(COM1_Int32MinValue), TCSupport.SerialPortRequirements.None);
-        retValue &= tcSupport.BeginTestcase(new TestDelegate(COM2_Neg1), TCSupport.SerialPortRequirements.None);
-        retValue &= tcSupport.BeginTestcase(new TestDelegate(COM3_0), TCSupport.SerialPortRequirements.None);
-        retValue &= tcSupport.BeginTestcase(new TestDelegate(COM4_Int32MaxValue), TCSupport.SerialPortRequirements.None);
-
-        _numErrors += tcSupport.NumErrors;
-        _numTestcases = tcSupport.NumTestcases;
-        _exitValue = tcSupport.ExitValue;
-
-        return retValue;
-    }
-
-
-    public bool COM1_9600()
+    
+    [Fact]
+    public void COM1_9600()
     {
         string portName = "COM1";
         int baudRate = 9600;
 
-        return VerifyCtor(portName, baudRate);
+        VerifyCtor(portName, baudRate);
     }
 
 
-    public bool COM2_14400()
+    [Fact]
+    public void COM2_14400()
     {
         string portName = "COM2";
         int baudRate = 14400;
 
-        return VerifyCtor(portName, baudRate);
+        VerifyCtor(portName, baudRate);
     }
 
 
-    public bool COM3_28800()
+    [Fact]
+    public void COM3_28800()
     {
         string portName = "COM3";
         int baudRate = 28800;
 
-        return VerifyCtor(portName, baudRate);
+        VerifyCtor(portName, baudRate);
     }
 
 
-    public bool COM4_57600()
+    [Fact]
+    public void COM4_57600()
     {
         string portName = "COM4";
         int baudRate = 57600;
 
-        return VerifyCtor(portName, baudRate);
+        VerifyCtor(portName, baudRate);
     }
 
 
-    public bool COM256_115200()
+    [Fact]
+    public void COM256_115200()
     {
         string portName = "COM256";
         int baudRate = 115200;
 
-        return VerifyCtor(portName, baudRate);
+        VerifyCtor(portName, baudRate);
     }
 
 
     //[] Error checking for PortName
-    public bool Empty_9600()
+    [Fact]
+    public void Empty_9600()
     {
-        string portName = String.Empty;
+        string portName = string.Empty;
         int baudRate = 9600;
 
-        return VerifyCtor(portName, baudRate, typeof(ArgumentException), ThrowAt.Set);
+        VerifyCtor(portName, baudRate, typeof(ArgumentException), ThrowAt.Set);
     }
 
 
-    public bool Null_14400()
+    [Fact]
+    public void Null_14400()
     {
         string portName = null;
         int baudRate = 14400;
 
-        return VerifyCtor(portName, baudRate, typeof(ArgumentNullException), ThrowAt.Set);
+        VerifyCtor(portName, baudRate, typeof(ArgumentNullException), ThrowAt.Set);
     }
 
 
-    public bool SlashSlash_28800()
+    [Fact]
+    public void SlashSlash_28800()
     {
         string portName = "\\\\";
```

Comment:
Sorry, missed this the first time through. All of these SlashSlash tests should be removed. The check was based on a misunderstanding of how paths work.